xbmc/platform/win32/DisplayUtilsWin32.cpp

```diff
@@ -0,0 +1,189 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "DisplayUtilsWin32.h"
+
+#include "ServiceBroker.h"
+#include "utils/StringUtils.h"
```

Comment:
#include "ServiceBroker.h"
#include "utils/StringUtils.h"

Not needed

Reply:
yes